kubernetes/helm/pinot/values.yaml

```diff
@@ -478,7 +486,9 @@ minionStateless:
 
   # Use extraEnv to add individual key value pairs as container environment variables.
   # ref: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
-  extraEnv: []
+  extraEnv:
+    - name: LOG4J_CONSOLE_LEVEL
+      value: info
```

Comment:
how about still keep default as warn and let the user choose to override this config by editing their own helm chart value override.